packages/api-client/src/server/electionSetup.ts

```diff
@@ -0,0 +1,43 @@
+import { JointKey } from '../models/jointKey';
+import ManifestPreview from '../models/manifestPreview';
+import { getAssignedGuardians } from './guardians';
+import { post } from '../utils/http';
+
+export const getManifestPreview = (): ManifestPreview => {
```

Comment:
Similar thoughts on isolating this to a mock data folder.

Reply:
Done